package.json

```diff
@@ -61,14 +61,14 @@
     "jquery-validation-unobtrusive": "3.2.6",
     "mark.js": "8.11.1",
     "minimist": "1.2.0",
-    "patternlab-node": "2.9.3",
+    "patternlab-node": "^2.12.0",
```

Comment:
Please use exact versioning for all modules in package.json

Reply:
Also when updating package.json I would guess that the package-lock.json also would be updated?

Reply:
reverted the version-changes